.github/workflows/tests.yml

```diff
@@ -117,10 +117,49 @@ jobs:
           CPUS: 2
           CREATE_VM: true
           KVM: true
+          SECUREBOOT: true
+        run: |
+          export RAW_IMAGE=$(find $PWD/build -name *.raw)
+          go run github.com/onsi/ginkgo/v2/ginkgo run --label-filter "raw-bootable" -v --fail-fast -r ./e2e
+  test-bootable-raw-bios:
+    runs-on: kvm
+    steps:
+      - uses: earthly/actions-setup@v1.0.13
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install deps
+        run: |
+          sudo apt-get update && sudo apt-get install -y ovmf libvirt-clients libvirt-daemon-system libvirt-daemon virtinst bridge-utils qemu-system-x86 qemu-system-x86 qemu-utils qemu-kvm acl udev
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+      - name: Build local image
+        uses: docker/build-push-action@v6
+        with:
+          push: false
+          load: true
+          builder: ${{ steps.buildx.outputs.name }}
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64
+          tags: auroraboot:latest
+      - name: Generate raw image
+        run: |
+          docker run --rm --privileged -v $PWD/build/:/output auroraboot:latest --debug --set "disable_http_server=true" --set "disable_netboot=true" --set "disk.bios=true" --set "disk.size=16000" --set "container_image=quay.io/kairos/ubuntu:24.04-core-amd64-generic-v3.2.3" --set "state_dir=/output"
+      - name: Test raw image is bootable
+        env:
+          USE_QEMU: true
+          MEMORY: 4000
+          CPUS: 2
+          CREATE_VM: true
+          KVM: true
+          SECUREBOOT: false
         run: |
-          # TODO: permissions stuff should be fixed on aurora side, the output dir should contain only
-          # there required output image/iso and not the leftovers from the build process
-          sudo chmod -R 777 build
-          #earthly +test-bootable-raw
           export RAW_IMAGE=$(find $PWD/build -name *.raw)
```

Comment:
should also be changed to `.efi` ?

Reply:
nope, this generates raw images without efi/bios in the name unfortunately. we cant do that without breaking our naming convention. Still bios images should not be the default

Reply:
Where is the old name expected? It would be better if we are consistent (intput flag matching the output file).

Reply:
what old name? this generates a raw image before runnig the tests agaisnt it. Like we do with the isos, we want to minimize the changes if (when) we bump versions) so this is the easier way to pass tot he test what image to use for the raw test. I dont get what the old name refores to? we used to generate shitty disk.raw images if thats what you refer to? But that was terrible and already removed since the first changes to Aurora disk raw generatin, now it produces identical names as the isos and such but with a different extension (.raw .vhd.raw, etc)